### PR TITLE
Feat/741 Known issues modal

### LINKED ISSUE #741: Add a Modal to introduce known issues on Token front end / aka release notes.
There are some issues with the staking process that will not have a quick fix. For these we agreed that 

- Headline things we want people to know about. Particularly in cases where values don't show what people expect (due to waiting epoch/blocks)
- Link to know issues (docs/blog post) 
- Summary of release notes

### PR DESCRIPTION
**Pull request by mattrussell36**
closes #741 

- Adds modal which appears on startup, once dismissed it doesn't appear again as we set a bool in local storage to tell if the user has already dismissed the modal

Screen Shot 2021-11-09 at 7 17 11 AM

### CODE REVIEW

**Comment by mattrussell36** [comment]
@candida-d You will probably want to change the language here.

@campbellssource This opens on startup regardless of page. Do you only want to show it on the staking page?

File: src/components/startup-modal/startup-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const MODAL_CLOSED_KEY = "vega_modal_closed";
+
+export const StartupModal = () => {
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = useLocalStorage(MODAL_CLOSED_KEY, true);
+
+  return (
+    <Overlay
+      isOpen={isOpen}
```
Comment (dexturr) [inline src/components/startup-modal/startup-modal.tsx:15]: We may need a flag to enable/disable this, as if we deploy this to prod right now with staking turned off then people are going to be forced to read that bug message even though they cannot stake.

**Review by dexturr — APPROVED** [review]

File: src/components/startup-modal/startup-modal.tsx (hunk 1 above)
Comment (mattrussell36) [reply src/components/startup-modal/startup-modal.tsx:15]: Good catch. Will update

File: src/i18n/translations/dev.json [hunk 2]
```diff
@@ -409,5 +409,8 @@
   "viewAllTranches": "View all tranches",
   "id": "ID",
   "noVersionFound": "Version could not be found, most likely your wallet version is <0.9.2 which is not supported.",
-  "unsupportedVersion": "Looks like you're running an outdated version of GoWallet. You're running {{version}} but {{requiredVersion}} is required."
+  "unsupportedVersion": "Looks like you're running an outdated version of GoWallet. You're running {{version}} but {{requiredVersion}} is required.",
```
Comment (candida-d) [inline src/i18n/translations/dev.json:412]:
```suggestion
  "unsupportedVersion": "Looks like you're running an outdated version of Vega Wallet. You're running {{version}} but {{requiredVersion}} is required.",
```

**Comment by candida-d** [comment]
@mattrussell36 I made some suggested changes, but had a quick chat with Michael as well and he suggested that we also link to the known issues in the footer (so the copy reflects that). Can that also be flagged?

**Pull request closed without merging**